lib/Sema/TypeChecker.h

```diff
@@ -1323,8 +1323,11 @@ class TypeChecker final : public LazyResolver {
   void computeAccessLevel(ValueDecl *D);
   void computeDefaultAccessLevel(ExtensionDecl *D);
 
+  /// Check the default arguments that occur within this value decl.
+  void checkDefaultArguments(ArrayRef<ParameterList *> params, ValueDecl *VD);
+
   virtual void resolveAccessControl(ValueDecl *VD) override {
-    validateAccessControl(VD);
+    validateAccessibility(VD);
```

Comment:
We don't use the term 'accessibility' since there's potential for confusion with the UI feature